Tests/Clappr_Tests/CoreTests.swift

```diff
@@ -95,6 +95,20 @@ class CoreTests: QuickSpec {
                 }
             }
 
+            describe("Core sharedData") {
+                context("when stores a value on sharedData") {
+
+                    beforeEach {
+                        core = CoreFactory.create(with: [:])
+                        core.sharedData["testKey"] = "testValue"
+                    }
+
```

Comment:
Could we create a test to ensure that the initial value is empty and after erase content the shared data was cleared?

Reply:
I did the same with the container tests! :)